deploy-board/deploy_board/webapp/env_views.py

```diff
@@ -561,6 +567,14 @@ def get(self, request, name, stage=None):
         return response
 
 
+def _gen_message_for_refreshing_cluster(lastClusterRefreshStatus, latest_succeeded_base_image_update_event):
+    if latest_succeeded_base_image_update_event != None:
+        if lastClusterRefreshStatus == None or lastClusterRefreshStatus["startTime"] == None or lastClusterRefreshStatus["startTime"] <= latest_succeeded_base_image_update_event["finish_time"]:
```

Comment:
```suggestion
    if latest_succeeded_base_image_update_event is not None:
        if lastClusterRefreshStatus is None or lastClusterRefreshStatus["startTime"] is None or lastClusterRefreshStatus["startTime"] <= latest_succeeded_base_image_update_event["finish_time"]:
```

Reply:
Alternative:
```suggestion
    if latest_succeeded_base_image_update_event is not None:
        if any([
          lastClusterRefreshStatus is None,
          lastClusterRefreshStatus["startTime"] is None,
          lastClusterRefreshStatus["startTime"] <= latest_succeeded_base_image_update_event["finish_time"],
        ])
```
https://docs.python.org/3.8/library/functions.html#any